### PR TITLE
Add batch movie ids endpoint

### PR DESCRIPTION
**Pull request by cricanr**
Added batch movieIds endpoint. QueryParameter parser.

### CODE REVIEW

File: app/query/QueryStringHelper.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            val maybeLimitParam = Try(paramStr(1)).toOption
+            maybeLimitParam.map { limitParam =>
+              val maybeLimitParams = Try(limitParam.split('=')).toOption
+              maybeLimitParams.flatMap(limitParams => limitParams.headOption)
+            }
+          }
+
+        }
+      }
+//        Try(movieIdsStr.split(",").map(_.toInt)).toOption)
```
Comment (crisanlucid) [inline app/query/QueryStringHelper.scala]: remove commented line

Comment (cricanr) [reply app/query/QueryStringHelper.scala]: Will do, thanks!

**Comment by cricanr** [comment]
> looks the same for me `param.contains("limit"))` and `param.contains("movieIds"))`
> 
> if yes, I will go with one method with more arguments

Well it is not really the same, one is single param, one is multi (array). I would refactor it as soon as we have another use case e.g. another single param or array param and make them generic, for now I guess is fine.

File: app/query/QueryStringHelper.scala (hunk 1 above)
Comment (cricanr) [reply app/query/QueryStringHelper.scala]: This is obsolete, no longer present in the last version.

**Review by crisanlucid — APPROVED** [review]

**Pull request merged**